README.md

```diff
@@ -727,6 +727,167 @@ Vue.component('courses-list', {
 
 :arrow_right: https://docs.angularjs.org/api/ng/service/$compile
 
+### Angular
+[Live Examples Here!](https://angular.io/generated/live-examples/lifecycle-hooks/eplnkr.html)
+
+#### [ngOnChanges()](https://angular.io/guide/lifecycle-hooks#onchanges)
+Respond when Angular (re)sets data-bound input properties. The method receives a `SimpleChanges` object of current and previous property values.
+
+Called before `ngOnInit()` and whenever one or more data-bound input properties change.
+```typescript
```

Comment:
Could you use `ts` shortcut? It's aligned with `js` / `jsx` and already in the linter config.